roles/openshift_node/tasks/apply_machine_config.yml

```
@@ -72,3 +72,17 @@
 - name: Reboot the host and wait for it to come back
   reboot:
   #  reboot_timeout: 600  # default, 10 minutes
+
+- name: Wait for nodes to report ready
+  command: >
+    oc get node {{ item | lower }}
```

```suggestion
    oc get node {{ ansible_nodename | lower }}
```

Currently it sets node ip (`oc get node 10.0.129.72`) instead of internal DNS name. `loop` should be removed IIUC

---

When testing this I was getting FQDN not IP.
```
oc get node ip-10-0-169-118.us-east-2.compute.internal
```
What environment you tested was using IP?
Also, `ansible_play_batch` in tasks/upgrade.yml would return the same value so are those also an issue in your environment?  I understand that the play is run `serial: 1` so the `loop` and `with_items` would really be necessary unless we were to ever make this play run on more than one host at a time.

---

That happens in CI - https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_openshift-ansible/11808/pull-ci-openshift-openshift-ansible-master-e2e-aws-scaleup-rhel7/842/build-log.txt

Seems loop is not necessary indeed, but lets keep it so that later on we could remove `serial` and speed up the scale up once its stable

---

I pushed a new commit to standardize on `ansible_nodename`.  Also, I found through testing that `ansible_play_batch` was using `inventory_hostname` so I've updated that task to be more specific about what is being used.  The `serial` is used on the upgrade.  If we need to speed up the upgrade we can come back to that later and implement the loops.